src/Compilers/VisualBasic/Test/Emit/EmitResourceUtil.vb

```diff
@@ -11,7 +11,9 @@ Imports System.Reflection
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
 
-    Public Class EmitResourceUtil
+    Public NotInheritable Class EmitResourceUtil
```

Comment:
> Public NotInheritable Class EmitResourceUtil [](http://example.com/codeflow?start=3&length=45)

Is this change and other similar changes necessary?

Reply:
Reverted everything under src/Compilers.